test/kerodon/unit/core.clj

```diff
@@ -335,6 +335,20 @@
       (test-fill-in-helper '([:label {:for "message-id"} "Message"]
                              [:textarea {:id "message-id" :name "message"}])
                            "Message" :#message-id "blah" "message=blah"))
+    (testing "multiple array inputs"
+      ;; not using label since doing so requires id, and having an id masks the problem
+      ;; from issue #47 when multiple inputs share the same name:
+      (let [state {:app (constantly :x)
+                   :enlive (parse [:form {:action "/" :method "post"}
+                                   [:input.array-input {:name "something"}]
```

Comment:
Nitpicks now, but I'll merge after these changes:

could you make the `:name` `something[]` so it's a bit more obvious why this is needed
could you either rename `nth-selector` to `nth-array-input` or make `:.array-input` into an argument, it's still a little tricky to follow at-a-glance.

let me know if you don't have time, I can pick these off myself at some point